dex/src/main/scala/com/wavesplatform/dex/error/MatcherError.scala

```diff
@@ -449,6 +449,10 @@ case object NonPositiveAssetRate extends MatcherError(rate, commonEntity, outOfB
 case class RateNotFound(theAsset: Asset)
     extends MatcherError(rate, commonEntity, notFound, e"The rate for the asset ${'assetId -> theAsset} was not specified")
 
+case object ApiKeyIsNotProvided extends MatcherError(auth, commonEntity, notProvided, e"API key is not provided")
```

Comment:
Suggest "API key is not provided in the configuration, please contact with the administrator"

Reply:
Done